lib/extensions/geolocation.js

```diff
@@ -30,8 +30,8 @@ async function setLocationWithLyft (udid, latitude, longitude) {
   } catch (e) {
     throw new Error(`'${LYFT_SET_LOCATION}' binary has not been found in your PATH. ` +
       'Please install it as "brew install lyft/formulae/set-simulator-location" by brew or ' +
```

Comment:
is formula name also changed?

Reply:
It seems no. Only the repository/org has changed, but the code owner is still lyft's person